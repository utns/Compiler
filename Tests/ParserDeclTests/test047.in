const
    c = 1;
var
    a: c;