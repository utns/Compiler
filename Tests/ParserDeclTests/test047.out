Syntax error. Expected '..' but ';' found: line = 4 pos = 9
